tests/conftest.py

```diff
@@ -1064,7 +1125,7 @@ def tmpvenv(tmpdir: py.path.local) -> Iterator[Tuple[py.path.local, py.path.loca
 
 
 @pytest.fixture
-def tmpvenv_active(deactive_venv, tmpvenv: py.path.local) -> Iterator[Tuple[py.path.local, py.path.local]]:
+def tmpvenv_active(deactive_venv, tmpvenv: py.path.local) -> Iterator[py.path.local]:
```

Comment:
There was indeed a mistake here but I think you got it the other way around.
```suggestion
def tmpvenv_active(deactive_venv, tmpvenv: Tuple[py.path.local, py.path.local]) -> Iterator[Tuple[py.path.local, py.path.local]]:
```

Reply:
If you don't like the tuple we could perhaps go with returning an instance of `PythonEnvironment` and `ActiveEnv` for `tmpvenv` and `tmpvenv_active` respectively.